### DISCUSSION TITLE
Getting all possible wyckoff positions

### DISCUSSION THREAD

**Discussion by stepan-tsirkin** [spglib/spglib#532] [General]
After determining the spacegroup of a cell, can spglib tell all possible Wyckoff positions,  in a form similar to Bilbao crystallographic server 

![{FCC65246-00AA-44E7-92F2-63864DB266D4}](https://github.com/user-attachments/assets/dfacb456-283a-46a9-9906-a1f6e16efa75)

But with x,y,z given in the reduced coordinates of a given primitive cell, not using the "reference cell"

Would it be hard to implement? any ideas?

**Comment by atztogo** [comment]
> can spglib tell all possible Wyckoff positions, in a form similar to Bilbao crystallographic server

No.

> Would it be hard to implement? any ideas?

User's input can be in non-standard setting or even not a primitive cell or the conventional unit cell. So we have to define an unconventional definition that would be misleading. Therefore, I think it is not a good idea and consider it is hard.
